hbase-server/src/test/java/org/apache/hadoop/hbase/procedure/SimpleRSProcedureManager.java

```diff
@@ -125,9 +125,8 @@ public Subprocedure buildSubprocedure(String name, byte[] data) {
 
     public SimpleSubprocedurePool(String name, Configuration conf) {
       this.name = name;
-      executor = new ThreadPoolExecutor(1, 1, 500,
-          TimeUnit.SECONDS, new LinkedBlockingQueue<>(),
-          Threads.newDaemonThreadFactory("rs(" + name + ")-procedure"));
+      executor = new ThreadPoolExecutor(1, 1, 500, TimeUnit.SECONDS, new LinkedBlockingQueue<>(),
```

Comment:
This is a bizarre instantiation, don't you think? keep-alive of 500 seconds? in a test? Maybe this can be just `Executors.newSingleThreadExecutor()`.